src/test/java/uk/ac/ebi/eva/pipeline/jobs/AnnotationJobTest.java

```diff
@@ -65,6 +66,9 @@
 public class AnnotationJobTest {
     private static final String MOCK_VEP = "/mockvep.pl";
     private static final String MONGO_DUMP = "/dump/VariantStatsConfigurationTest_vl";
+    private static final String OUTPUT_DIR_ANNOTATION = "/tmp/";
```

Comment:
Use a `PipelineTemporaryFolderRule` instead of a hardcoded path. Applies to other classes using the `outputDirAnnotation(...)` method.